pandas/indexes/category.py

```diff
@@ -468,6 +468,24 @@ def take(self, indices, axis=0, allow_fill=True, fill_value=None):
                                            na_value=-1)
         return self._create_from_codes(taken)
 
+    def map(self, mapper):
+        """
```

Comment:
Left it as it is, because ``Categorical.map`` needs some supplementary info. Using ``Appender`` doesn't make it very simple.